src/pixel-display-impl_.toit

```diff
@@ -77,6 +77,17 @@ abstract class PixelDisplay implements Window:
   inner-width: return driver_.width
   inner-height: return driver_.width
 
+  /**
+    By default, the display is rendered in patches that have
+      a max size of 2-8k, depending on the bits per pixel.  On devices
+      with a large amount of memory this patch size can be increased,
+      which may improve performance.
+    During rendering, the display may allocate extra buffers for
+      multiple color components, transparency, and clipping, so the
+      effective memory use may be higher than this number.
+  */
+  max-patch-size/int := 2000
```

Comment:
Maybe make it `:= ?` and always set it in the constructor?
Otherwise you might get misled by just looking here (and not realizing that some displays, like the two-color one) set it to a different value.

You could also have `TWO-COLOR-DEFAULT-PATCH-SIZE`... constants.